RELEASE.md

```diff
@@ -16,6 +16,7 @@
 ## Bug fixes and other changes
 * Use default `False` value for rich logging `set_locals`, to make sure credentials and other sensitive data isn't shown in logs.
 * The Kedro IPython extension now surfaces errors when it cannot load a Kedro project.
+* Relaxed `delta-spark` upper bound to allow compatibility with Spark 3.x.x
```

Comment:
Does it works for all Spark 3.x.x? I don't know this well but seems we only support `delta-spark` >=1.0 before so spark 3.0.x is out of the picture.
![image](https://user-images.githubusercontent.com/18221871/184144198-41c21aea-43ee-48a9-bdbe-b56029177bc8.png)

https://docs.delta.io/2.0.0/releases.html

Reply:
Correct - my mistake, the user was specifically asking for Spark 3.2 support